eZ/Publish/API/Repository/SearchService.php

```diff
@@ -32,7 +32,7 @@
      * @param array $fieldFilters - a map of filters for the returned fields.
      *        Currently supports: <code>array("languages" => array(<language1>,..), "useAlwaysAvailable" => bool)</code>
      *                            useAlwaysAvailable defaults to true to avoid exceptions on missing translations
-     * @param boolean $filterOnUserPermissions if true only the objects which is the user allowed to read are returned.
+     * @param boolean $filterOnUserPermissions if true only the objects which the user is allowed to read are returned.
```

Comment:
Fixed in https://github.com/ezsystems/ezpublish-kernel/commit/f42b07aaf3500688ac383f502eabb256e05a9d83